tests/integration_tests/functional/test_vhost_user_block.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
we already have some virtio block tests in `test_drive.py`. Shall we name this file in a similar way, eg `test_drive_vhost_user.py`?

Reply:
renamed